apps/crc_proposal_end.py

```diff
@@ -16,36 +16,26 @@
 class CrcProposalEnd(BaseParser):
     """Display the end date for an account's current CRC proposal."""
 
-    banking_db_path = 'sqlite:////ihome/crc/bank/crc_bank.db'
-
     def __init__(self) -> None:
         """Define arguments for the command line interface"""
 
         super(CrcProposalEnd, self).__init__()
-
         default_group = Shell.run_command("id -gn")
-        self.add_argument(
-            'account', default=default_group, nargs='?',
-            help=f'the name of the Slurm account [default: {default_group}]')
+        help_text = f"SLURM account name [defaults to your primary group: {default_group}]"
+        self.add_argument('account', nargs='?', default=default_group, help=help_text)
 
-    def get_proposal_end_date(self, account: str) -> datetime:
+    def get_proposal_end_date(self, account: str) -> str:
```

Comment:
This function is simple enough that it can be merged with the main `app_logic` method.